packages/gspell.rb

```diff
@@ -42,8 +43,6 @@ class Gspell < Package
   depends_on 'vala' => :build
   depends_on 'zlibpkg' # R
 
-  ENV['XML_CATALOG_FILES'] = "#{CREW_PREFIX}/etc/xml/catalog"
```

Comment:
Add to libxml2 package in `#{CREW_PREFIX}/etc/env.d/libxml2`?

Reply:
Already in `docbook_xml`'s env.d file:`XML_CATALOG_FILES=/usr/local/etc/xml/catalog`

Reply:
```
$ crew whatprovides /etc/xml/catalog
```
Returns nothing?

Reply:
```ruby
require 'package'

class Docbook_xml < Package
  description 'Meta package for all versions of docbook_xml'
  homepage 'http://www.docbook.org'
  version '5.1-2-2'
  license 'MIT'
  compatibility 'all'

  is_fake

  unless File.exist?("#{CREW_PREFIX}/etc/env.d/docbook_xml")
    FileUtils.mkdir_p "#{CREW_PREFIX}/etc/env.d/"
    @env = <<~DOCBOOK_XML_EOF
      # Docbook_xml configuration
      XML_CATALOG_FILES=#{CREW_PREFIX}/etc/xml/catalog
    DOCBOOK_XML_EOF
    File.write("#{CREW_PREFIX}/etc/env.d/docbook_xml", @env)
  end

  depends_on 'xmlcatmgr'
  depends_on 'docbook_xml412'
  depends_on 'docbook_xml42'
  depends_on 'docbook_xml43'
  depends_on 'docbook_xml44'
  depends_on 'docbook_xml45'
  depends_on 'docbook_xml50'
  depends_on 'docbook_xml51'
  # These are probably needed at the same time too.
  depends_on 'docbook_xsl'
  depends_on 'docbook_xsl_nons'
end
```

How should we deal with this?

Reply:
We could make `docbook_xml` a real package...

Reply:
This is already there from `docbook_xsl`:
```bash
chronos@cheekon-x86_64 /usr/local/lib/crew/packages $ crew whatprovides env.d/docbook_xml
docbook_xsl: /usr/local/etc/env.d/docbook_xml

Total found: 1
```

Reply:
The docbook packages generate the files for the etc/xml location in any case...

Reply:
(`/usr/local/etc/xml/catalog` is like the mandb database.... it needs to be generated and thus won't be in a package.)